File: Android/aryanmishra29_AryanMishra_2125csme1029_2/GettingStarted/Print_Fibonacci_till_n.java
import java.util.*;
public class Print_Fibonacci_till_n {
    public static void printFib(int n){
        int a=0;
        int b=1;
        if(n==1) System.out.println(a);
        else if(n==2){
            System.out.println(a);
            System.out.println(b);
        }
        else{
            System.out.println(a);
            System.out.println(b);
            for(int i=2;i<n;i++){
                int c=a+b;
                System.out.println(c);
                a=b;
                b=c;
            }
        }
    }
  
  public static void main(String[] args) {
      // write your code here
      Scanner sc=new Scanner(System.in);
      int n=sc.nextInt();
      printFib(n);
   }
}

File: Android/khushi1502_KhushiSachdev_2125cse1077_2/GettingStarted/PrintFibonaccitillN.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main(int argc, char **argv){
    int n;
    cin >> n;
  //write your code here

    int i=0,j=1,dig;
    
    while (n>0){
        dig = n % 10;
        n = n / 10;
        i += j * pow(10, dig - 1);
        j++;
    }

    cout << (i) << endl;
}